### PR TITLE
Submission for Assign. 3 'Trigrams' of Lesson04

### PR DESCRIPTION
**Pull request by stanslov7**

### CODE REVIEW

**Comment by stanslov7** [comment]
Added submission for except_exercise.py of lesson05 to pull #1182

File: students/stan_slov7/lesson04/trigrams.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    print("Total words length:",len(words),"\n")
+    
+    word_pairs = build_trigrams(words)
+    print("Building trigrams based on word pairs...\n")
+    
+    new_text = build_text(word_pairs)
+    print("Printing generated text...\n")
+    print(new_text)
+    print()
+    print("-------End of Generated Text-------\n")
```
Comment (charlesrobison) [inline students/stan_slov7/lesson04/trigrams.py:113]: Looks good and works well!

**Review by charlesrobison — APPROVED** [review]
Nice job on this - looks good and works well.

Ready to merge

**Comment by stanslov7** [comment]
Added mailroom3.py for submission - (Assign 4) for Lesson05, commits for this got added to this pull request

**Comment by stanslov7** [comment]
Added mailroom4.py and test_mailroom4.py for lesson06 submission to this pull

**Comment by stanslov7** [comment]
Added circle_class.py and test_circle_class.py submissions for lesson08 assignment to the pull request

**Comment by stanslov7** [comment]
Added the files for html_render Assignment for lesson 07 to pull

**Comment by stanslov7** [comment]
Added submission files for mailroom5 object oriented version for lesson 09.
Finalized submissions for the course.

**Comment by natasha-aleksandrova** [comment]
@charlesrobison can you review new additions please?

**Pull request merged**